lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -293,6 +293,9 @@ def extract_student(student, features):
             if include_enrollment_mode:
                 student_dict['enrollment_mode'] = enrollment_mode
 
+        preferred_lang = profile.language_proficiencies.values_list('code', flat=True)
+        student_dict['language'] = preferred_lang[0] if preferred_lang else profile.language
```

Comment:
Are you sure that `preferred_lang[0]` return current(latest) value? Maybe we need to use last element of the list?

Reply:
Student can chose only one language in profile or account page